### PR TITLE
Upgrade to NPM 7

### PR DESCRIPTION
**Pull request by asemanh**
I tried to update our script. Something I'm not sure is if it's ok to delete the package-lock.json for shared-asset

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "clean": "./node_modules/.bin/lerna run clean --stream",
     "build": "./node_modules/.bin/lerna run build --stream",
     "serve": "./node_modules/.bin/http-server -a 127.0.0.1 -o /packages/modelviewer.dev -c-1",
-    "update:package-lock": "rm -f ./package-lock.json && npm i && ./node_modules/.bin/lerna run update:package-lock --stream"
+    "update:package-lock": "rm -f ./package-lock.json && npm i && ./node_modules/.bin/lerna run update:package-lock --stream",
```
Comment (elalish) [inline package.json:15]: I have no problem with removing the shared-assets package-lock, but we should also remove its package.json.

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
...
+    "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap && npm run clean"
```
Comment (elalish) [inline package.json:16]: I am still getting `npm WARN old lockfile The package-lock.json file was created with an old version of npm` when I run update:package-lock from your branch. Do you get that if you pull it to your other machine?

File: package.json (hunk 1 above)
Comment (asemanh) [reply package.json:15]: I think we still need the package.json for the script

File: package.json (hunk 2 above)
Comment (asemanh) [reply package.json:16]: Sounds like you're not seeing that if you run upgrade?

File: package.json (hunk 1 above)
Comment (elalish) [reply package.json:15]: It seems to work, so we can skip it for now, but let's do a follow-on to clean up shared-assets.

File: package.json (hunk 2 above)
Comment (elalish) [reply package.json:16]: Indeed, let's simplify so we only include the working script, under the old name.

**Review by elalish — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (asemanh) [reply package.json]: Yeah, when we run "npm install", it does call "prepare" script [aligns with npm documentation] which ends up fetching the khornos assets. And I agree that's outside of the scope of this task

File: package.json (hunk 2 above)
Comment (asemanh) [reply package.json]: Just sent an updated version

**Pull request merged**